packages/scene-composer/src/components/panels/TopBar.tsx

```diff
@@ -39,7 +41,7 @@ export const TopBar: FC = () => {
   }, [nodeMap]);
 
   const hasCameraView = cameraItems.length > 0;
-  const showTopBar = hasCameraView;
+  const showTopBar = hasCameraView && !matterportSdk;
```

Comment:
logically, you probably want to do 
```
  const showCameraView = cameraItems.length > 0 && !matterportSdk;
  const showTopBar = showCameraView;
```
since you are hiding the camera view specifically, not the whole top bar

Reply:
Both are same as per the current implementation. I will add back the camera view feature once Matterport SDK has fix for this issue.